providers/gov/pubmedcentral/pmc/normalizer.py

```
@@ -0,0 +1,243 @@
+                'organization'
+            )
+        ))
+
+    class Extra:
+        correspondence = OneOf(
+            ctx.record.metadata.article.front['article-meta']['author-notes']['corresp']['email'],
+            ctx.record.metadata.article.front['article-meta']['author-notes']['corresp'],
+            Static(None)
+        )
+        journal = ctx.record.metadata.article.front['journal-meta']['journal-title-group']['journal-title']
+        in_print = Try(RunPython('get_print_information', ctx.record.metadata.article.front['article-meta']))
+        issn = (RunPython('get_issns', ctx.record.metadata.article.front['journal-meta']['issn']))
+        doi = RunPython('get_list_element', ctx.record.metadata.article.front['article-meta']['article-id'],
+                        '@pub-id-type', 'doi')
+        pubmed_id = (RunPython('get_list_element', ctx.record.metadata.article.front['article-meta']['article-id'],
+                        '@pub-id-type', 'pmcid'))
+
+        copyright = OneOf(
+            ctx.record.metadata.article.front['article-meta']['permissions']['copyright-statement']['#text'],
+            ctx.record.metadata.article.front['article-meta']['permissions']['copyright-statement'],
+            Static(None)
+        )
+        copyright_year = Try(ctx.record.metadata.article.front['article-meta']['permissions']['copyright-year'])
+        epub_date = RunPython('get_year_month_day', ctx.record.metadata.article.front['article-meta']['pub-date'], 'epub')
+        ppub_date = RunPython('get_year_month_day', ctx.record.metadata.article.front['article-meta']['pub-date'], 'ppub')
+
+    def get_contributors(self, ctx, type):
+        results = []
+
+        if type == 'person':
+            for contributor in ctx:
+                try:
+                    contributor['name']
+                    results.append(contributor)
+                except:
```

Don't use bare except block, they will catch all exceptions, even keyboard interrupts.
Instead just catch `Exception`

However if you're just trying to catch a `KeyError`  just catch that or do a check ahead of time

```python
if 'name' in contributor:
...
```